test/controllers/v1/relay_group_membership_controller_test.exs

```diff
@@ -48,6 +46,20 @@ defmodule Cog.V1.RelayGroupMembershipControllerTest do
     assert member["name"] == relay.name
   end
 
+  test "relay group includes assigned bundles", %{authed: requestor} do
+    bundle = bundle("foo")
+    relay_group = relay_group("test-relay-group-1")
+    assign_bundle_to_group(relay_group.id, bundle.id)
+    conn = api_request(requestor, :get, "/v1/relay_groups/#{relay_group.id}")
+    fetched = json_response(conn, 200)["relay_group"]
+    assert relay_group.id == fetched["id"]
+    assert relay_group.name == fetched["name"]
+    assert length(fetched["bundles"]) == 1
+    member = Enum.at(fetched["bundles"], 0)
+    assert member["id"] == bundle.id
+    assert member["name"] == bundle.name
```

Comment:
I like this. Unless I'm missing something though, we don't need to worry about the match operator and can just use `bundle.id` and friends on the LHS where we are creating the map that we compare with `fetched`. Am I missing something, @christophermaier?